### PR TITLE
Trello config

### LINKED ISSUE #6: Fix trello configuration options
Trello uses `consumerKey` and `consumerSecret` instead of `clientID` and `clientSecret`. Fix this to be consistent with other auth providers and update documentation.

### PR DESCRIPTION
**Pull request by NicerHugs**
fixes #6

### CODE REVIEW

File: config/auth.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ module.exports = {
 		redirect: '/dashboard'
 	},
 	trello: {
-		consumerKey: '{{ Trello Consumber Key }}',
```
Comment (alarner) [inline config/auth.js:32]: @NicerHugs small typo here "Consumber"

File: lib/auth/adapters/trello.js [hunk 2]
```diff
@@ -2,19 +2,18 @@ var OAuth1Strategy = require('passport-oauth1');
...
-				}
+	OAuth1Strategy.call(this, Object.assign({
+		requestTokenURL: 'https://trello.com/1/OAuthGetRequestToken',
+		accessTokenURL: 'https://trello.com/1/OAuthGetAccessToken',
+		userAuthorizationURL: 'https://trello.com/1/OAuthAuthorizeToken',
+		sessionKey: 'trello',
+		consumerKey: options.clientID,
+		consumerSecret: options.clientSecret
+	}, options), (req, accessToken, refreshToken, profile, done) => {
+		console.log(accessToken);
```
Comment (alarner) [inline lib/auth/adapters/trello.js]: Please remove console.log @NicerHugs

**Comment by alarner** [comment]
@NicerHugs Just a few small tweaks. I tested it out and it works great! I even found some bugs in the email code I had previously written, which are now fixed up on master.

**Comment by NicerHugs** [comment]
@alarner pr updated

**Pull request merged**